localenv/cloud-nine-wallet/dbinit.sql

```diff
@@ -9,3 +9,7 @@ ALTER DATABASE cloud_nine_wallet_auth OWNER TO cloud_nine_wallet_auth;
 CREATE USER happy_life_bank_backend WITH PASSWORD 'happy_life_bank_backend';
 CREATE DATABASE happy_life_bank_backend;
 ALTER DATABASE happy_life_bank_backend OWNER TO happy_life_bank_backend;
+
+CREATE USER happy_life_bank_auth WITH PASSWORD 'happy_life_bank_auth';
+CREATE DATABASE happy_life_bank_auth;
+ALTER DATABASE happy_life_bank_auth OWNER TO happy_life_bank_auth;
```

Comment:
@mkurapov do we add new lines at the end of our text files?

Reply:
good q, I think usually we do. Updated